Annotations/AddCustomStampInCustomizedButton/MainWindow.xaml.cs

```diff
@@ -2,7 +2,9 @@
 using Syncfusion.Windows.PdfViewer;
 using System;
 using System.Collections.Generic;
+using System.IO;
 using System.Linq;
+using System.Runtime.InteropServices;
```

Comment:
Check whether this is necessary for all the samples?

Reply:
Un necessary reference removed in sample completed